internal/integratedservices/integratedservicedriver/transport_http.go

```diff
@@ -45,28 +48,30 @@ func RegisterHTTPHandlers(endpoints Endpoints, router *mux.Router, errorHandler
 		options...,
 	))
 
-	router.Methods(http.MethodGet).Path("/{featureName}").Handler(kithttp.NewServer(
+	serviceRouter := router.Path(fmt.Sprintf("/{%s}", integratedServiceNameParamKey)).Subrouter()
```

Comment:
I'd rather stick to a single `serviceName` key and use that even in the legacy routes. It does not break backwards compatibility.